templates/Menus/admin.html.twig

```diff
@@ -37,6 +37,11 @@
                     Report Research Group Dataset Status
                 </a>
             </li>
+            <li class="first odd">
+                <a href="{{ path('pelagos_app_ui_reportresearchgroupdatasetstatus_datasetmonitoringreport') }}">
+                    Dataset Report for Research Group
+                </a>
+            </li>
             <li class="first even"><a href="{{ path('pelagos_app_ui_gomrireport_default') }}">GOMRI Dataset Report(v1)</a></li>
```

Comment:
Even tho these classes don't do anything, but this would have be called middle instead of first.